srcpkgs/dfu-util/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'dfu-util'
+pkgname=dfu-util
+version=0.9
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="libusb-devel"
+short_desc="USB Device Firmware Class programmer"
+maintainer="Foxlet <foxlet@furcode.co>"
+homepage="http://dfu-util.sourceforge.net/"
+license="GPL-2.0-only"
```

Comment:
see the man page:
````
LICENCE
       dfu-util is covered by the GNU General Public License (GPL), version 2
       or later.
````

Reply:
The website itself only mentions GPLv2, which is newer than when the manpage was written.

Reply:
all right.